resources/clients/react/elros-pam/src/pam-api.ts

```diff
@@ -50,16 +50,14 @@ export const s3Copy = async (bucketName: string, config: PamApiConfig) => {
 
 export const initializeDownload = async (
   labels: string[],
-  notify: string,
   config: PamApiConfig
 ) => {
   const response = await fetch(
-    `${import.meta.env.VITE_API_GATEWAY_BASE_URL}restore`,
+    `${import.meta.env.VITE_API_GATEWAY_BASE_URL}download`,
```

Comment:
(nit) I'd rather do this with a URL, but that's a future us problem

Reply:
Like a URL object?